EIPS/eip-1690.md

```diff
@@ -0,0 +1,178 @@
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+
+A standard that allows the owner of a contract to completely remove it.
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+
+For removing a smart contract the solidity function `selfdestruct(address);` needs to be called from inside the Contract.
+This function allowes to provide a address where the ether currently owned by the smart contract will be transferred before deleting the contract.
+It should be called from outside the contract by the owner only using a public function.
+
+## Motivation
+<!--The motivation is critical for EIPs that want to change the Ethereum protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the EIP solves. EIP submissions without sufficient motivation may be rejected outright.-->
+
+To completely remove a smart contract that is e.g. not needed anymore or was created by mistake, the functionality needs to be implemented in the contract.
+This EIP should standardize this functionality.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+
+This standard uses the EIP-173 Contract Ownership Standard to handle the owner management.
+It provides two functions to kill the Smart Contract.
+Both functions must only be executed by the owner.
+One of the function allows to send the contract's ether to any provided address, the other should just transfer the ether to the owner.
+
+## Rationale
+<!--The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+
+A similar implementation of this standard is also used as example implementation for Smart Contracts, the [Greeter](https://www.ethereum.org/greeter), on the official Ethereum Website.
```

Comment:
I think this link is dead now.

Reply:
I found an article on medium.com that still uses the greeter example

```suggestion
A similar implementation of this standard is also used as example implementation for Smart Contracts, the [Greeter](https://medium.com/shokone/https-medium-com-shokone-building-a-hello-world-smart-contract-on-ethereum-f303c7d05f0), in an article on medium.com.
```